pupil_src/shared_modules/video_capture/uvc_backend.py

```diff
@@ -175,35 +177,56 @@ def verify_drivers(self):
             if proc.returncode == 2:
                 ids_present += 1
                 ids_to_install.append(id)
-        work_dir = os.getcwd()
+
         if ids_present > 0:
+            logger.warning("Updating drivers, please wait...")
+
+            # NOTE: libwdi in PupilDrvIns.exe cannot deal with unicode characters in the
+            # temporary path where the drivers will be installed. Check for non-ascii in
+            # current working directory and use C:\Windows\Temp as fallback.
             try:
-                os.mkdir(os.path.join(work_dir, "win_drv"))
-            except FileExistsError:
-                pass
-            cmd_str = ""
-            for id in ids_to_install:
-                # using ''' here to be able to use both " and ' without escaping
-                # Note: ArgumentList needs quotes ordered this way (' outer, " inner),
-                # otherwise it won't work
-                cmd_str += f'''Remove-Item "{work_dir}\win_drv" -recurse -Force; '''
-                cmd_str += (
-                    f'''Start-Process PupilDrvInst.exe -Wait -Verb runas'''
-                    f''' -WorkingDirectory "{work_dir}"'''
-                    f''' -ArgumentList '--vid {id[0]} --pid {id[1]} --desc "{id[2]}" --vendor "Pupil Labs" --inst'; '''
+                str(Path.cwd().resolve()).encode("ascii")
+            except UnicodeEncodeError:
+                temp_path = Path("C:\\Windows\\Temp")
+                if not temp_path.exists():
+                    raise RuntimeError(
+                        "Username contains Unicode characters and default Temp folder "
```

Comment:
This is also technically not correct. It could also be the path to the bundle, not only the username. I'll adjust this as well.